ext4: export iov_shorten from kernel for ext4's use
From: Eric Sandeen <sandeen@redhat.com>

Export iov_shorten() from kernel so that ext4 can
truncate too-large writes to bitmapped files.

Signed-off-by: Eric Sandeen <sandeen@redhat.com>

---

---
 fs/read_write.c |    2 ++
 1 file changed, 2 insertions(+)

Index: linux-2.6.24-rc3/fs/read_write.c
===================================================================
--- linux-2.6.24-rc3.orig/fs/read_write.c	2007-11-16 21:16:36.000000000 -0800
+++ linux-2.6.24-rc3/fs/read_write.c	2007-12-05 11:41:33.000000000 -0800
@@ -451,6 +451,8 @@ unsigned long iov_shorten(struct iovec *
 	return seg;
 }
 
+EXPORT_SYMBOL(iov_shorten);
+
 ssize_t do_sync_readv_writev(struct file *filp, const struct iovec *iov,
 		unsigned long nr_segs, size_t len, loff_t *ppos, iov_fn_t fn)
 {
